bot/core/event.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+
+from typing import Any
+
+import attrs
+
+BAD_ATTRS = ["app", "shard"]
+
+
+def filter(attr: attrs.Attribute[Any], value: Any) -> bool:
+    if attr.name.startswith("_"):
+        return False
+    if attr.name in BAD_ATTRS:
+        return False
+    return True
+
+
+async def handle_event(event: object) -> None:
+    data = attrs.asdict(event, recurse=True, filter=filter)
+    print(data)
```

Comment:
This isn't gonna work imo. I think events should ALL be serialized by hand because we need pick and choose what parts we want.

Reply:
I think we want all parts relating to the model's data